UICatalog/Scenarios/Notepad.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Terminal.Gui;
+using static UICatalog.Scenario;
+
+namespace UICatalog.Scenarios {
+
+	[ScenarioMetadata (Name: "Notepad", Description: "Multi tab text editor")]
+	[ScenarioCategory ("Controls")]
+	class Notepad : Scenario {
+
+		TabView tabView;
+		Label lblStatus;
+
+		private int numbeOfNewTabs = 1;
+
+		public override void Setup ()
+		{
+			Win.Title = this.GetName ();
+			Win.Y = 1; // menu
+			Win.Height = Dim.Fill (1); // status bar
+			Top.LayoutSubviews ();
+
+			var menu = new MenuBar (new MenuBarItem [] {
+				new MenuBarItem ("_File", new MenuItem [] {
+					new MenuItem ("_New", "", () => New()),
+					new MenuItem ("_Open", "", () => Open()),
```

Comment:
When I choose this two bad things happen:

1. The `Ok` button in the fIle open dialog doesn't say `Ok`
2. If I ht `tab` 5 times the app crashes.

I think you're doing something wrong in your File Open dialog calling code. I can't reproduce this using the `Editor` scenario.

Reply:
Note "Save As" seems to work fine.

Reply:
Think it was calling the blank constructor that was causing this problem.  I resolved it by using the strings constructor.

Reply:
@tznind please check if it will also throws exception in the `UICatalog` `All Views Tester`.

Reply:
Ok have figured out what I did wrong.  I used `FileDialog` instead of `OpenDialog`.  But it does seem that there is a bug when using `FileDialog()` blank constructor that results in the Open button having no text and crashing when Tabbed to.  Do you want me to raise a seperate issue for that?

Reply:
Yes, I think you should.